File: app/src/main/java/team/triplog/presentation/activity/MoreUserActivity.java
package team.triplog.presentation.activity;

import android.os.Bundle;
import android.view.MenuItem;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.appcompat.widget.Toolbar;

import com.bumptech.glide.Glide;
import com.bumptech.glide.load.resource.bitmap.RoundedCorners;
import com.bumptech.glide.request.RequestOptions;

import team.triplog.R;
import team.triplog.data.local.User;

public class MoreUserActivity extends BaseActivity {
    private Toolbar toolbar;
    private TextView textUserName;
    private TextView textUserCode;
    private TextView textUserAccount;
    private ImageView viewUserProfile;
    private User user;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_more_user);

        init();
        setData();
        setUi();
    }

    private void init() {
        toolbar = findViewById(R.id.toolbar);
        textUserName = findViewById(R.id.text_user_name);
        textUserCode = findViewById(R.id.text_user_code);
        textUserAccount = findViewById(R.id.text_user_account);
        viewUserProfile = findViewById(R.id.view_user_profile);

        setSupportActionBar(toolbar);
        if (getSupportActionBar() != null) {
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
            getSupportActionBar().setDisplayShowHomeEnabled(true);
            getSupportActionBar().setDisplayShowTitleEnabled(false);
        }
    }

    private void setData() {
        user = realm.where(User.class).findFirst();
    }

    private void setUi() {
        textUserName.setText(user.getName());

        Glide.with(this)
                .load(user.getImage())
                .apply(RequestOptions.bitmapTransform(new RoundedCorners(16)))
                .thumbnail(0.1f)
                .into(viewUserProfile);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            finish();
            return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/team/triplog/presentation/activity/MoreUserActivity.kt
package team.triplog.presentation.activity

import android.os.Bundle
import android.view.MenuItem
import com.bumptech.glide.Glide
import com.bumptech.glide.load.resource.bitmap.RoundedCorners
import com.bumptech.glide.request.RequestOptions
import kotlinx.android.synthetic.main.activity_more_user.*
import team.triplog.R
import team.triplog.data.local.User
import team.triplog.presentation.base.BaseActivity

class MoreUserActivity : BaseActivity() {
    private var user: User? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_more_user)

        init()
        setData()
        setUi()
    }

    private fun init() {
        setSupportActionBar(toolbar)
        if (supportActionBar != null) {
            supportActionBar!!.setDisplayHomeAsUpEnabled(true)
            supportActionBar!!.setDisplayShowHomeEnabled(true)
            supportActionBar!!.setDisplayShowTitleEnabled(false)
        }
    }

    private fun setData() {
        // TODO : Room 으로 변경하기 위해 주석처리
//        user = realm?.where(User::class.java)?.findFirst()
    }

    private fun setUi() {
        text_user_name.text = user?.name
        Glide.with(this)
            .load(user?.image)
            .apply(RequestOptions.bitmapTransform(RoundedCorners(16)))
            .thumbnail(0.1f)
            .into(view_user_profile)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
            return true
        }
        return super.onOptionsItemSelected(item)
    }
}